### PR TITLE
Add msgpack extension

### PR DESCRIPTION
**Pull request by etienneroudeix**

### CODE REVIEW

File: layers/msgpack/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+RUN pecl install --force msgpack
+RUN cp `php-config --extension-dir`/msgpack.so /tmp/msgpack.so
+
+# Build the final image from the lambci image that is close to the production environment
```
Comment (shouze) [inline layers/msgpack/Dockerfile:7]:
```suggestion
```

File: layers/msgpack/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+RUN pecl install --force msgpack
```
Comment (shouze) [inline layers/msgpack/Dockerfile:4]: I'm pretty sure that you don't need `--force` option.
```suggestion
RUN pecl install msgpack
```

**Review by shouze — COMMENTED** [review]
only 2 suggestions excepting that LGTM

**Review by shouze — APPROVED** [review]
LGTM

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**